### PR TITLE
 vsmile_cart.xml: New WORKING software list additions 

### PR DESCRIPTION
**Pull request by clawgrip**
-------------------------
Assepoester - De wonderwereld van Assepoester (NL) [TeamEurope]
Aladdin - De wonderwereld van Aladdin (NL) [TeamEurope]
Scooby-Doo! - Griezelen in het Pretpark (NL) [TeamEurope]
Alfabet Pretpark (NL) [TeamEurope]

New NOT_WORKING software list additions
-------------------------
Ratatouille-Remy's Nieuwe Recepten (NL) [TeamEurope]
Winnie de Poeh - De wonderwereld van Poeh (NL) [TeamEurope]

### CODE REVIEW

File: hash/vsmile_cart.xml [hunk 1]
```diff
@@ -256,7 +258,7 @@ Game cartridges
 +========+===================+================================================================================================+
 |        | 80-092200(US)     | Aladdin - Aladdin's Wonders of the World                                                       |
 |   XX   | 80-092201(IT)     | Aladdin - Il magico mondo di Aladdin                                                           |
-|        | 80-092202(NL)     | Aladdin - De Wunderwereld van Aladdin                                                          |
+|   XX   | 80-092202(NL)     | Aladdin - De Wunderwereld van Aladdin                                                          |
```
Comment (happppp) [inline hash/vsmile_cart.xml]: Wonderwereld?

File: hash/vsmile_cart.xml [hunk 2]
```diff
@@ -567,6 +569,23 @@ Game cartridges
 		</part>
 	</software>
 
+	<software name="aladdinnl" cloneof="aladdini" supported="yes">
+		<description>Disney's Aladdin - De winderwereld van Aladdin (Netherlands)</description>
```
Comment (happppp) [inline hash/vsmile_cart.xml]: another Wonderwereld

File: hash/vsmile_cart.xml (hunk 1 above)
Comment (clawgrip) [reply hash/vsmile_cart.xml]: Fixed!

File: hash/vsmile_cart.xml (hunk 2 above)
Comment (clawgrip) [reply hash/vsmile_cart.xml]: Fixed!

**Pull request merged**